source/github-helpers/index.ts

```diff
@@ -21,9 +21,15 @@ Tested on isRepoTree, isBlame, isSingleFile, isEditFile, isSingleCommit, isCommi
 Example tag content on public repositories: https://github.com/sindresorhus/refined-github/commits/branch-or-commit/even/with/slashes.atom
 Example tag content on private repositories https://github.com/private/private/commits/master.atom?token=AEAXKWNRHXA2XJ2ZWCMGUUN44LM62
 */
-export const getCurrentBranch = (): string => {
+export const getCurrentBranch = (): string | undefined => {
 	// .last needed for #2799
-	return new URL(select.last<HTMLLinkElement>('[type="application/atom+xml"]')!.href)
+	const atom = select.last<HTMLLinkElement>('[type="application/atom+xml"]');
+	// The atom URL is not available on `isIssue` #3641
+	if (!atom) {
+		return undefined;
```

Comment:
```suggestion
		return;
```

Doesn’t work? Or just `void` as the type?

Reply:
I’m on mobile now but it sounds strange that there’s no atom feed on issues at all

Reply:
I agree! This was working all along.